konfera/models/order.py

```diff
@@ -1,10 +1,14 @@
 from django.db import models
+from django.utils import timezone
 from django.utils.translation import ugettext_lazy as _
 
 
+AWAITING = 'awaiting_payment'
+PAID = 'paid'
+
 ORDER_CHOICES = (
-    ('awaiting_payment', _('Awaiting payment')),
-    ('paid', _('Paid')),
+    (AWAITING, _('Awaiting payment')),
+    (PAID, _('Paid')),
     ('expired', _('Expired')),
     ('cancelled', _('Cancelled')),
```

Comment:
since we have started to use constants in this list, please update all strings into constants, same way you did with AWAITING and PAID.